examples/_debug/widgets/geojson-category-data-view-all-modes.html

```diff
@@ -0,0 +1,175 @@
+            </p>
+            <p>
+              <span class="as-caption">Limit</span>
+              <input id="limit" class="as-input" type="number" value="5" />
+            </p>
+            <p>
+              <span class="as-caption">Operation</span>
+              <as-dropdown default-text="Operation" can-clear="false" selected-option="count"></as-dropdown>
+            </p>
+            <p>
+              <span class="as-caption">Operation column</span>
+              <input id="operationColumn" class="as-input" type="text" value="pop_est" />
+            </p>
+            <button class="as-btn as-btn--secondary" onclick="applyChanges();">
+              APPLY
+            </button>
+          </div>
+        </div>
+        <div class="as-panel as-panel--vertical as-panel--top as-panel--left as-bg--ui-01">
+          <div class="as-panel__element as-p--16 as-body" style="max-width: fit-content; padding-top: 0px;">
+            <as-category-widget id="category-widget-remote" class="as-p--16" heading="Cities (remote)">
+            </as-category-widget>
+            <as-category-widget id="category-widget-remote-filtered" class="as-p--16" heading="Cities (remote-filtered)">
+            </as-category-widget>
+            <as-category-widget id="category-widget-local" class="as-p--16" heading="Cities (local)">
+            </as-category-widget>
+          </div>
+        </div>
+    </main>
+  </div>
+
+  <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.js"></script>
+  <script src="https://unpkg.com/deck.gl@8.2.0-beta.1/dist.min.js"></script>
+  <script src="https://libs.cartocdn.com/airship-components/v2.3/airship.js"></script>
+
+  <script src="/dist/umd/index.min.js"></script>
+
+  <script>
+    const operationElem = document.querySelector('as-dropdown');
+    operationElem.options = [
+      { text: 'SUM', value: 'sum' },
+      { text: 'COUNT', value: 'count' },
+      { text: 'AVG', value: 'avg' },
+      { text: 'MAX', value: 'max' },
+      { text: 'MIN', value: 'min' }
+    ];
+    const widgetRemoteElement = document.getElementById(
+      'category-widget-remote'
+      );
+    const widgetRemoteFilteredElement = document.getElementById(
+      'category-widget-remote-filtered'
+      );
+    const errorContainer = document.getElementById('errorContainer');
+
+    let categoryRemote;
+    let categoryRemoteFiltered;
+    let categoryLocal;
+
+    async function initialize() {
+      const deckMap = carto.viz.createMap();
+
+      const geojson = await fetch('https://public.carto.com/api/v2/sql?api_key=default_public&q=select* from ne_50m_admin_0_countries&format=geojson').then(response => response.json())
+      const countriesLayer = new carto.viz.Layer(geojson);
+      await countriesLayer.addTo(deckMap);
+
+      categoryRemote = new carto.dataview.Category(
+        countriesLayer.source,
+        'continent',
+        { operation: 'count', operationColumn: 'pop_est', mode: 'remote' }
+      );
```

Comment:
Some comments, also for @manmorjim. Can't we already pass just a `countriesLayer` instead of countriesLayer.source` for this?

Also, the 'count' examples are maybe not the best ones, as it doesn't really need an `operationColumn' (that would probably confuse our "future us")

Reply:
> Can't we already pass just a `countriesLayer` instead of countriesLayer.source` for this?
 
Yes, we can :smiley:

Reply:
I have updated this example from the original one which is using SUM and layers